lib/internal/modules/esm/module_job.js

```diff
@@ -1,10 +1,14 @@
 'use strict';
 
 const {
+  ArrayPrototypeJoin,
   ObjectSetPrototypeOf,
   PromiseAll,
   SafeSet,
   SafePromise,
+  StringPrototypeIncludes,
+  StringPrototypeMatch,
+  StringPrototypeSplit
```

Comment:
is there a reason node's eslint doesn't require trailing object commas?

Reply:
It doesn't seem intentional.

https://github.com/nodejs/node/blob/06cdf08b337a2aef127f18f2f6ffab74aa7a18f6/.eslintrc.js#L83

Reply:
I think the reason is that it would require changing too many lines at once to enable it.